src/api/resources/empathicVoice/resources/chat/client/Socket.ts

```diff
@@ -28,16 +28,16 @@ export class ChatSocket{
         this.socket = socket;
         this.readyState = socket.readyState;
 
-        this.socket.addEventListener('open', this.handleOpen);
-        this.socket.addEventListener('message', this.handleMessage);
-        this.socket.addEventListener('close', this.handleClose);
-        this.socket.addEventListener('error', this.handleError);
+        this.socket.addEventListener("open", this.handleOpen);
```

Comment:
can we keep single quotation marks for JS/TS?

Reply:
we can update the prettier config, i just left it how it was before